components/wifi/esp8266-driver/mbed_lib.json

```diff
@@ -21,6 +21,23 @@
             "help": "RESET pin for the modem, defaults to Not Connected",
             "value": null
         },
+        "pwr": {
+            "help": "POWER pin for the modem, defaults to Not Connected",
+            "value": null
+        },
+        "power-on-polarity": {
+            "help": "Polarity of power-on for the modem. 0 means 0/1 for power on/off; 1 means 1/0 for power on/off.",
+            "options": [0, 1],
+            "value": 0
+        },
+        "power-on-time-ms": {
+            "help": "Power-on time in ms for the modem",
```

Comment:
Clarify this to delay _after_ powering on modem.

Reply:
Modified